deployment/docker/rules/alerts-vmalert.yml

```diff
@@ -83,3 +83,14 @@ groups:
           summary: "vmalert instance {{ $labels.instance }} is failing to send notifications to Alertmanager"
           description: "vmalert instance {{ $labels.instance }} is failing to send alert notifications to \"{{ $labels.addr }}\".
             Check vmalert's logs for detailed error message."
+
+      - alert: RemoteWriteDroppingData
```

Comment:
It looks like a valid rule, but duplicates with rule `RemoteWriteErrors`(only differing in`for` and I don't know why)) since their metrics change simultaneously, so user will receive two alerts with the same messages. Is this expected?

Reply:
Yes, the new alerting rule has clear title explaining consequences of what's happening.
RemoteWrite errors could be perceived as something temporary, while dropped data is immediately perceived as permanent effect. 
tl;dr; I expect this rule to be treated a way more seriously than RW errors.